x-pack/plugins/observability_solution/investigate_app/public/pages/details/investigation_details_page.tsx

```diff
@@ -7,7 +7,7 @@
 
 import { EuiButton, EuiButtonEmpty, EuiText } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import { alertOriginSchema } from '@kbn/investigation-shared';
+import { alertOriginSchema } from '@kbn/investigation-shared/src/schema';
```

Comment:
How about you do following in `packages/kbn-investigation-shared/src/index.ts`

`export * from './schema';` 

And then do following in this file?
`import { alertOriginSchema } from '@kbn/investigation-shared/`